lib/rack/multipart/parser.rb

```diff
@@ -213,6 +213,8 @@ def initialize(boundary, tempfile, bufsize, query_parser)
 
         @sbuf = StringScanner.new("".dup)
         @body_regex = /(?:#{EOL}|\A)--#{Regexp.quote(boundary)}(?:#{EOL}|--)/m
+        @body_regex_at_end = /#{@body_regex}\z/m
```

Comment:
I don't think `@body_regex_at_end` is used by this code, so it should be added.

Reply:
Oops, it came with the cherry pick by accident. I removed it.